app/src/main/res/values/strings.xml

```diff
@@ -338,6 +338,7 @@
   <string name="yesterday">yesterday</string>
   <string name="return_to_topic">Return to topic</string>
   <string name="explanation">Explanation:</string>
+  <string name="if_two_items_are_equal_merge_them">If two items are equal, merge them.</string>
```

Comment:
Suggest: ``drag_and_drop_interaction_group_merge_hint``.

Also, is there a corresponding issue on Oppia web tracking adding this string to the web frontend? If not, please file one.

Reply:
done